packages/admin/src/pages/super-admin-dashboard/index.page.tsx

```diff
@@ -67,80 +115,99 @@ const SuperAdminDashboard = ({
         <div className="govuk-width-container">
           <main className="govuk-main-wrapper">
             <div className="govuk-grid-row">
-              <div className={`${styles.sidebar} govuk-grid-column-one-third`}>
-                <h2 className="govuk-heading-l">Manage users</h2>
-                {/* counter */}
-                <p className="govuk-body">
-                  We’ve found <strong>{userCount}</strong> users
-                </p>
-                {/* filter button */}
-                <div className={styles['top-controls']}>
-                  <Button text="Clear all filters" isSecondary />
-                </div>
+              <FlexibleQuestionPageLayout
+                csrfToken={csrfToken}
+                fieldErrors={fieldErrors}
+                fullPageWidth
+                formAction={formAction}
+              >
+                <div
+                  className={`${styles.sidebar} govuk-grid-column-one-third`}
+                >
+                  <h2 className="govuk-heading-l">Manage users</h2>
+                  {/* counter */}
+                  <p className="govuk-body">
+                    We’ve found <strong>{userCount}</strong> users
+                  </p>
+
+                  {/* filter button */}
+                  <div className={styles['top-controls']}>
+                    <Button text="Clear all filters" isSecondary />
+                  </div>
+
+                  <Checkboxes
+                    useOptionValueAsInputValue
+                    questionTitle="Role"
+                    fieldName="roles"
+                    defaultCheckboxes={previousRoles || []}
+                    titleSize="m"
+                    options={roles
+                      .filter((role) => role.name !== 'FIND')
+                      .map((role) => ({
+                        label: role.label,
+                        value: role.id,
+                      }))}
```

Comment:
should we maybe just show `FIND` for now even if it doesn't mean anything yet, rather than requiring a code change to display it later?